### DISCUSSION TITLE
How to register plugins with Chart primevue

### DISCUSSION THREAD

**Discussion by inane** [primefaces/community#413] [PrimeVue]
Hello

I need to use some plugins from Chart.js. But I am not being able to integrate in my vue project.

Any idea to integrate them?   Specifically I need to use zoom-plugin and streaming.plugin.

`          <Chart
            type="line"
            :data="setChartData"
            :options="setChartOptions"
            :plugins="[zoomPlugin, ChartStreaming]"
          />`
          
          ZoomPlugin is working but ChartStreaming not... 
          
          Here the options:
          
          `plugins: {
      legend: {
        labels: {
          color: textColor
        }
      },
      zoom: {
        pan: {
          enabled: true,
          mode: 'x'
        },
        zoom: {
          wheel: {
            enabled: true
          },
          pinch: {
            enabled: true
          },
          mode: 'xy'
        }
      },
      streaming: {
        duration: 20000
      }
    },
    scales: {
      x: {
        type: 'realtime',
        // Change options only for THIS AXIS
        realtime: {
          onRefresh: function (chart) {
            chart.data.datasets.forEach(function (dataset) {
              dataset.data.push({
                x: Date.now(),
                y: Math.random()
              })
            })
          }
        }
      },
      y: {
        beginAtZero: true,
        ticks: {
          color: textColorSecondary
        }
      }
    }
`

Thank you in advance

**Comment by FlipWarthog** [comment]
See this related discussion, you may be running into a similar limitation: https://github.com/orgs/primefaces/discussions/187
